adapters/rubicon/rubicon.go

```diff
@@ -648,20 +658,11 @@ func (a *RubiconAdapter) MakeRequests(request *openrtb.BidRequest, reqInfo *adap
 
 				// set user.ext.tpid
 				if len(userExt.Eids) > 0 {
-					tpIds := make([]rubiconExtUserTpID, 0)
-					for _, eid := range userExt.Eids {
-						if eid.Source == "adserver.org" {
-							uids := eid.Uids
-							if len(uids) > 0 {
-								uid := uids[0]
-								if uid.Ext != nil && uid.Ext.RtiPartner == "TDID" {
-									tpIds = append(tpIds, rubiconExtUserTpID{Source: "tdid", UID: uid.ID})
-								}
-							}
-						}
-					}
-					if len(tpIds) > 0 {
-						userExtRP.TpID = tpIds
+					if tpIds, segments, errors := getTpIdsAndSegments(userExt.Eids); len(errors) > 0 {
+						errs = append(errs, errors...)
```

Comment:
Probably wanna add a `continue` after this line as well

Reply:
Yes, thanks!